mkdocs.yml

```diff
@@ -21,7 +21,7 @@ pages:
           - 'Intro': 'data-management/quantumleap/readme.md'
     - 'IoT Services Enablement': 
         - 'IDAS': 'iot-services/readme.md'
-        - 'IoT Agent JSON': 'iot-services/iotagent-json/readme.md'
+#        - 'IoT Agent JSON': 'iot-services/iotagent-json/readme.md'
```

Comment:
oops, I guess unintended

Reply:
follow up question, did you use any specific tool to apply these changes?

Reply:
follow-up question... did you use any specific tool to apply these changes? I've seen some good linters able to detect but not yet to adjust.

Reply:
it's not a mistake, for some reason, the mkdocs generate errors on this inclusion.

Reply:
i used manually atom, but probably there are things that lint MD automatically... to be checked

Reply:
Oops, sorry for the double-comment, some connectivity issues here.
So, we can't just comment it out, it's an index in the docs hehe. Try changing the filename from `readme.md` to `README.md`, does that make the lint happier?